packages/app/src/search/searchers/base.searcher.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { Index, SearchResults } from "elasticlunr"
+import { IndexStorage } from "App/index-storage/types"
+
+export abstract class BaseSearcher {
+  constructor(public index: IndexStorage) {}
+
+  // AUTO DISABLED - fix me if you like :)
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  abstract search(query: string): any[] | undefined
```

Comment:
```suggestion
  abstract search(query: string): unknown[] | undefined
```

it's possible to add an easily `uknown` or generic type for this case? 🤔